examples/cleanup-controller/README.md

```diff
@@ -0,0 +1,40 @@
+# Cleanup Controller
+
+The cleanup controller removes PersistentVolumes and PersistentVolumeClaims that reference deleted Nodes.
+
+## Overview
+
+Pods using a Local Persistent Volume are always scheduled to the same Node as the Local PV it uses (as opposed to if they were using a HostPath volume, for instance). When Nodes fail while having a Local PV attached to them, Pods using the Local PV become stuck since they can't be scheduled to a deleted Node. In other words, the Local PV and its corresponding PVC aren’t cleaned up when a Node becomes unavailable and so each of them (PV/PVC/Pods) becomes stuck. This results in a degraded or unavailable workload. This controller aims to clean up the stale Local PVs and their bound PVCs after Node deletion, allowing workloads to automatically recover.
+
+## Design
+
+There are two separate processes, a **CleanupController** and a **Deleter**, running to delete stale resources:
```

Comment:
processes or threads?

Reply:
I suppose both? I put processes because the controller is running to delete PVCs and the Deleter is running to delete PVs so they are separate. Though, they would be running on different threads.

Reply:
They're running in the same process but as two separate go routines.

Reply:
I've changed it to "routines."